Python/bytecodes.c

```diff
@@ -1411,12 +1411,13 @@ dummy_func(
                 goto error;
             }
 
-            if (!Py_CHECKWRITE(cell)){
+            if (!Py_CHECKWRITE(cell)) {
                 format_exc_notwriteable(tstate, frame->f_code, oparg);
                 goto error;
             }
 
             PyCell_SET(cell, NULL);
+            Py_REGIONREMOVEREFERENCE(cell, oldobj);
```

Comment:
I am wondering if we should have
```suggestion
            Py_DECREF_WITH_REGION(oldobj, cell);
```
Or perhaps REMOVEREFERENCE should do a decrement.  This is a broader question than this PR.